examples/boto3/service-2.sdk-extras.json

```diff
@@ -12,6 +12,17 @@
             "input":{"shape":"DeleteBucketNotificationConfigurationRequest"},
             "documentationUrl":"https://docs.ceph.com/docs/master/radosgw/s3/bucketops/#delete-notification",
             "documentation":"<p>Deletes the notification configuration from the bucket.</p>"
+        },
+        "GetUsageStats":{
+            "name":"GetUsageStats",
+            "http":{
+                "method":"GET",
+                "requestUri":"/?usage",
+                "responseCode":200
+            },
+            "output": {"shape": "GetUsageStatsOutput"},
+            "documentationUrl":"https://docs.ceph.com/docs/master/radosgw/s3/",
```

Comment:
please point to the actual documentation of the feature?
is this a standard S3 feature?

Reply:
nope, this is an RGW only extension, looks like we haven't documented it at https://docs.ceph.com/docs/master/radosgw/s3/serviceops/

Reply:
> nope, this is an RGW only extension, looks like we haven't documented it at https://docs.ceph.com/docs/master/radosgw/s3/serviceops/

so i guess we should... can you add this as part of this PR?

Reply:
will do